src/main/java/com/palantir/gradle/versions/FixLegacyJavaConfigurationsPlugin.java

```diff
@@ -44,41 +44,30 @@ public final void apply(Project project) {
             return;
         }
 
-        Configuration unifiedClasspath = project
-                .getRootProject()
+        Configuration unifiedClasspath = project.getRootProject()
                 .getConfigurations()
                 .findByName(VersionsLockPlugin.UNIFIED_CLASSPATH_CONFIGURATION_NAME);
         Preconditions.checkNotNull(
-                unifiedClasspath,
-                "FixLegacyJavaConfigurationsPlugin must be applied after VersionsLockPlugin");
+                unifiedClasspath, "FixLegacyJavaConfigurationsPlugin must be applied after VersionsLockPlugin");
```

Comment:
:+1: this is nice - laying out method invocation args on one line where possible rather than two!